### ISSUE TITLE
Copy Failed Error

### ISSUE THREAD

**Issue by gaggle331** [HaveAGitGat/Tdarr#389]
**Describe the bug**
After transcoding has completed and the server needs to copy the file from cache to the final resting location, I get an error like this: `Error: ENOENT: no such file or directory, lstat '/mnt/transcode_cacheUp.2009.PROPER.1080p.BluRay.H264.AAC-RARBG-TdarrCacheFile-Vb7Gey12j.mkv'` It appears that there is a `/` missing between `transcode_cache` and the filename. I've tried adding the `/` in both the node's pathTranslator config and in the cache config for the library but it still gives the same error.

**To Reproduce**
Run any set of transcodes with the node running on Windows and server running in docker on Unraid.

**Expected behavior**
The newly transcoded file should be copied and overwrite the original file.

**Screenshots**
Error in the Staging Section
![image](https://user-images.githubusercontent.com/12423051/109436022-adc2fa80-79da-11eb-9f3e-5419096b366d.png)

After clicking the `i` on the error
![image](https://user-images.githubusercontent.com/12423051/109436039-c9c69c00-79da-11eb-9fa0-85cdc56fed82.png)

Tdarr Server Library Cache Config 
![image](https://user-images.githubusercontent.com/12423051/109436055-e06cf300-79da-11eb-8e53-38d5bc820038.png)

Tdarr Node pathTranslator Config (Windows 10) 
![image](https://user-images.githubusercontent.com/12423051/109436072-f2e72c80-79da-11eb-87bc-5c76100d8bf5.png)

Tdarr Server Docker Config (Unraid) 
![image](https://user-images.githubusercontent.com/12423051/109436130-448fb700-79db-11eb-9532-fbc8852fdac2.png)

**Please provide the following information:**
- Config files  [can be found in /app/configs/ when using Docker or in the /configs folder next to Tdarr_Updater if not using Docker]

- Log files  [can be found in /app/logs/ when using Docker or in the /logs folder next to Tdarr_Updater if not using Docker]

-Worker error [can be found on the 'Tdarr' tab by pressing the 'i' button on a failed item in the staged file section or in the transcode error section at the bottom]

 - OS: Windows 10
 - Browser Chrome
 - Version 2.00.06

**Additional context**
None

**Comment by HaveAGitGat** [comment]
Are you able to try the `dev` container on unraid and the 2.00.07 pre-release for the Windows node? It should download it if you manually specify 2.00.07 in the updater config:

```
{
  "platform_arch": "default",
  "modules": {
    "Tdarr_Desktop": {
      "enabled": true,
      "requiredVersion": "latest",
      "currentVersion": "2.00.07"
    },
    "Tdarr_Node": {
      "enabled": true,
      "requiredVersion": "latest",
      "currentVersion": "2.00.07"
    },
    "Tdarr_Server": {
      "enabled": true,
      "requiredVersion": "latest",
      "currentVersion": "2.00.07"
    },
    "Tdarr_WebUI": {
      "enabled": true,
      "requiredVersion": "latest",
      "currentVersion": "2.00.07"
    }
  }
}
```

Check the console output on the node (not the log.txt) and it should show exactly what the path translation is doing. This should help determine at which stage it's going wrong. Thanks.
![image](https://user-images.githubusercontent.com/43864057/109927461-8f078100-7cc4-11eb-9389-c72b4c0998cc.png)

**Comment by gaggle331** [comment]
I updated to the `dev` container on unraid and the 2.00.07 pre-release for the Windows node but it doesn't output that path translation in the console output. Is there a way to ensure my versions are correct?

**Comment by HaveAGitGat** [comment]
Make sure to have verbose logs enabled on the logs tab.

**Comment by gaggle331** [comment]
Got it. Here's what the console output shows. 
![image](https://user-images.githubusercontent.com/12423051/110184881-273d5d00-7dce-11eb-970f-2306e7e51ade.png)

**Comment by HaveAGitGat** [comment]
Hi thanks for that. Are you able to post the full terminal output?

I'm trying to see at which point the slash is lost between the `W:` and `Onward...`

![image](https://user-images.githubusercontent.com/43864057/110198803-5cd95900-7e55-11eb-974a-5de196d2bd93.png)

**Comment by gaggle331** [comment]
```
[2021-03-06T10:36:37.418] [INFO] Tdarr_Node - [3.312s]Plugin update finished
[2021-03-06T10:36:54.189] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:54.194] [INFO] Tdarr_Node - verbose:Worker cPn1geIJp launched
[2021-03-06T10:36:54.195] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:54.198] [INFO] Tdarr_Node - verbose:Worker pLkqqK2KCv launched
[2021-03-06T10:36:54.199] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:54.204] [INFO] Tdarr_Node - verbose:Worker NFDhCqe_0h launched
[2021-03-06T10:36:54.205] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:54.209] [INFO] Tdarr_Node - verbose:Worker YSuD4V2j9p launched
[2021-03-06T10:36:54.579] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:54.581] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:54.583] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:54.584] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:54.585] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:54.585] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:54.585] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:54.589] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:54.590] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:54.588] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:54.600] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:54.600] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:54.601] [INFO] Tdarr_Node - verbose:Item received
[2021-03-06T10:36:54.602] [INFO] Tdarr_Node - verbose:Requesting item
 ---------Translating-serverToNode---------
[2021-03-06T10:36:54.604] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:54.604] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:54.605] [INFO] Tdarr_Node - verbose:Exit request
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
[2021-03-06T10:36:54.608] [INFO] Tdarr_Node - verbose:Worker exit request received
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
[2021-03-06T10:36:54.609] [INFO] Tdarr_Node - verbose:Exiting
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
[2021-03-06T10:36:54.610] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:36:54.610] [INFO] Tdarr_Node - verbose:Worker exit request received
 Obj translated from:
[2021-03-06T10:36:54.612] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T10:36:54.613] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:36:54.613] [INFO] Tdarr_Node - verbose:Worker exit request received
{
  _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  originalLibraryFile: {
    _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}
[2021-03-06T10:36:54.615] [INFO] Tdarr_Node - verbose:Exiting
 To:
{
  _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  originalLibraryFile: {
    _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}

 ----------------------------------
 ---------Translating-serverToNode---------
 PT (/mnt/transcode_cache) -> (W:):/mnt/transcode_cache -> W:
 PT (/mnt/tv) -> (V:):/mnt/tv -> V:
 Obj translated from:
{ cache: '/mnt/transcode_cache', folder: '/mnt/tv' }
 To:
{ cache: 'W:', folder: 'V:' }

 ----------------------------------
[2021-03-06T10:36:54.626] [INFO] Tdarr_Node - verbose:Worker exited
[2021-03-06T10:36:54.627] [INFO] Tdarr_Node - verbose:Worker exited
[2021-03-06T10:36:54.628] [INFO] Tdarr_Node - verbose:Worker exited
 ---------Translating-nodeToServer---------
 PT (V:) -> (/mnt/tv):V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> /mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (V:) -> (/mnt/tv):V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> /mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 Obj translated from:
{
  file: {
    _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}
 To:
{
  file: {
    _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}

 ----------------------------------
[2021-03-06T10:36:56.190] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:56.193] [INFO] Tdarr_Node - verbose:Worker 8El2DlSGL launched
[2021-03-06T10:36:56.194] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:56.198] [INFO] Tdarr_Node - verbose:Worker GAC6GDjhFm launched
[2021-03-06T10:36:56.199] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:56.203] [INFO] Tdarr_Node - verbose:Worker p1Go2UUjqg launched
[2021-03-06T10:36:56.203] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:36:56.208] [INFO] Tdarr_Node - verbose:Worker XBZE7lBa35 launched
[2021-03-06T10:36:56.579] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:56.579] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:56.582] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:56.583] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:56.584] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:56.585] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:56.589] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:36:56.590] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:36:56.589] [INFO] Tdarr_Node - verbose:Worker exit request received
[2021-03-06T10:36:56.597] [INFO] Tdarr_Node - verbose:Worker exit request received
[2021-03-06T10:36:56.597] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T10:36:56.597] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T10:36:56.595] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:56.598] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:56.601] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:56.602] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:56.603] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:56.604] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:56.608] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:36:56.608] [INFO] Tdarr_Node - verbose:Worker exit request received
[2021-03-06T10:36:56.608] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:36:56.609] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T10:36:56.610] [INFO] Tdarr_Node - verbose:Worker exit request received
[2021-03-06T10:36:56.611] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T10:36:56.613] [INFO] Tdarr_Node - verbose:Worker exited
[2021-03-06T10:36:56.616] [INFO] Tdarr_Node - verbose:Worker exited
[2021-03-06T10:36:56.623] [INFO] Tdarr_Node - verbose:Worker exited
 ---------Translating-serverToNode---------
[2021-03-06T10:36:56.627] [INFO] Tdarr_Node - verbose:Worker exited
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7 -> V:/The Flash (2014)/Season 7
 Obj translated from:
{
  _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  file: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  meta: {
    SourceFile: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    Directory: '/mnt/tv/The Flash (2014)/Season 7'
  }
}
 To:
{
  _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  meta: {
    SourceFile: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    Directory: 'V:/The Flash (2014)/Season 7'
  }
}

 ----------------------------------
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
{
  _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  container: 'mkv',
  ffProbeRead: 'success',
  createdAt: 1615052216258,
  lastPluginDetails: 'none',
  bit_rate: 6780351.734175755,
  statSync: {
    dev: 49,
    mode: 33188,
    nlink: 1,
    uid: 99,
    gid: 100,
    rdev: 0,
    blksize: 4096,
    ino: 1535785,
    size: 2145232095,
    blocks: 4189912,
    atimeMs: 1614822480703.108,
    mtimeMs: 1614823606226.341,
    ctimeMs: 1614823663578.247,
    birthtimeMs: 0,
    atime: '2021-03-04T01:48:00.703Z',
    mtime: '2021-03-04T02:06:46.226Z',
    ctime: '2021-03-04T02:07:43.578Z',
    birthtime: '1970-01-01T00:00:00.000Z'
  },
  file_size: 2045.8527517318726,
  ffProbeData: { streams: [ [Object], [Object], [Object] ] },
  meta: {
    SourceFile: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    errors: [],
    Duration: 2531.116,
    DefaultDuration: 0.032,
    ExifToolVersion: 12.1,
    FileName: "The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    Directory: 'V:/The Flash (2014)/Season 7',
    FileSize: '2046 MB',
    FileModifyDate: {
      year: 2021,
      month: 3,
      day: 4,
      hour: 2,
      minute: 6,
      second: 46,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:04 02:06:46+00:00'
    },
    FileAccessDate: {
      year: 2021,
      month: 3,
      day: 4,
      hour: 1,
      minute: 48,
      second: 0,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:04 01:48:00+00:00'
    },
    FileInodeChangeDate: {
      year: 2021,
      month: 3,
      day: 4,
      hour: 2,
      minute: 7,
      second: 43,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:04 02:07:43+00:00'
    },
    FilePermissions: 'rw-r--r--',
    FileType: 'MKV',
    FileTypeExtension: 'mkv',
    MIMEType: 'video/x-matroska',
    EBMLVersion: 1,
    EBMLReadVersion: 1,
    DocType: 'matroska',
    DocTypeVersion: 2,
    DocTypeReadVersion: 2,
    TimecodeScale: '1 ms',
    MuxingApp: 'libebml v1.3.3 + libmatroska v1.4.4',
    WritingApp: "mkvmerge v9.0.1 ('Obstacles') 64bit",
    DateTimeOriginal: {
      year: 2021,
      month: 3,
      day: 3,
      hour: 23,
      minute: 21,
      second: 19,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:03 23:21:19Z'
    },
    VideoCodecID: 'V_MPEG4/ISO/AVC',
    VideoFrameRate: 23.976,
    TrackLanguage: 'und',
    ImageWidth: 1920,
    ImageHeight: 1080,
    DisplayWidth: 1920,
    DisplayHeight: 1080,
    AudioCodecID: 'A_AC3',
    AudioSampleRate: 48000,
    AudioChannels: 6,
    TrackNumber: 3,
    TrackType: 'Subtitle',
    TrackDefault: 'No',
    CodecID: 'S_TEXT/UTF8',
    ImageSize: '1920x1080',
    Megapixels: 2.1
  },
  mediaInfo: { '@ref': '', track: [ [Object], [Object], [Object], [Object] ] },
  hasClosedCaptions: false,
  DB: 'CSwLJ736O',
  bumped: false,
  HealthCheck: '',
  TranscodeDecisionMaker: '',
  processingStatus: false,
  fileMedium: 'video',
  infoLog: '',
  video_codec_name: 'h264',
  audio_codec_name: '',
  video_resolution: '1080p',
  lastHealthCheckDate: 0,
  lastTranscodeDate: 0,
  history: '',
  oldSize: 0,
  newSize: 0,
  tPosition: 0,
  hPosition: 0
}
{
  container: '.mkv',
  FFmpegMode: true,
  handBrakeMode: false,
  infoLog: '☑ Multi channel audio already exists in AC3\n' +
    '☑ No audio processing necessary\n' +
    '☒ Transcoding to HEVC using NVidia NVENC\n' +
    'Encoder configuration:\n' +
    '• Original Bitrate: 6466\n' +
    '• Target Bitrate: 3694\n' +
    '• Minimum Bitrate: 2585\n' +
    '• Maximum Bitrate: 4802\n' +
    '\n' +
    '☑ No subtitle processing necessary',
  processFile: false,
  preset: '-c:v h264_cuvid,-map 0 -map -0:d -c:v hevc_nvenc -rc:v vbr_hq -cq:v 19 -b:v 3694k -minrate 2585k -maxrate 4802k -bufsize 6466k -spatial_aq:v 1 -rc-lookahead:v 32 -c:a copy -c:s copy -max_muxing_queue_size 4096',
  reQueueAfter: false,
  lastPluginDetails: {
    source: 'Community',
    id: 'Tdarr_Plugin_drdd_standardise_all_in_one',
    number: '2/2'
  },
  reason: 'requireGPU',
  ffmpegMode: true
}
 ---------Translating-nodeToServer---------
 PT (V:) -> (/mnt/tv):V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> /mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 Obj translated from:
{
  obj: {
    _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}
 To:
{
  obj: {
    _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
[2021-03-06T10:36:56.694] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:36:56.695] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:36:56.695] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:36:56.698] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:36:56.698] [INFO] Tdarr_Node - verbose:Worker exit request received
[2021-03-06T10:36:56.699] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T10:36:56.710] [INFO] Tdarr_Node - verbose:Worker exited
[2021-03-06T10:37:00.201] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:37:00.205] [INFO] Tdarr_Node - verbose:Worker 4s_2pL62Z launched
[2021-03-06T10:37:00.520] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:37:00.523] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:37:00.523] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:37:00.535] [INFO] Tdarr_Node - verbose:Item received
 ---------Translating-serverToNode---------
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 Obj translated from:
{
  _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  originalLibraryFile: {
    _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}
 To:
{
  _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  originalLibraryFile: {
    _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}

 ----------------------------------
 ---------Translating-serverToNode---------
 PT (/mnt/transcode_cache) -> (W:):/mnt/transcode_cache -> W:
 PT (/mnt/tv) -> (V:):/mnt/tv -> V:
 Obj translated from:
{ cache: '/mnt/transcode_cache', folder: '/mnt/tv' }
 To:
{ cache: 'W:', folder: 'V:' }

 ----------------------------------
 ---------Translating-nodeToServer---------
 PT (V:) -> (/mnt/tv):V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> /mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (V:) -> (/mnt/tv):V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> /mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 Obj translated from:
{
  file: {
    _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}
 To:
{
  file: {
    _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    file: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}

 ----------------------------------
[2021-03-06T10:37:02.210] [INFO] Tdarr_Node - verbose:Launching worker
[2021-03-06T10:37:02.213] [INFO] Tdarr_Node - verbose:Worker -KkSTWBHT launched
 ---------Translating-serverToNode---------
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 PT (/mnt/tv) -> (V:):/mnt/tv/The Flash (2014)/Season 7 -> V:/The Flash (2014)/Season 7
 Obj translated from:
{
  _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  file: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  meta: {
    SourceFile: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    Directory: '/mnt/tv/The Flash (2014)/Season 7'
  }
}
 To:
{
  _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  meta: {
    SourceFile: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    Directory: 'V:/The Flash (2014)/Season 7'
  }
}

 ----------------------------------
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
{
  _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  file: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
  container: 'mkv',
  ffProbeRead: 'success',
  createdAt: 1615052222072,
  lastPluginDetails: 'none',
  bit_rate: 6780351.734175755,
  statSync: {
    dev: 49,
    mode: 33188,
    nlink: 1,
    uid: 99,
    gid: 100,
    rdev: 0,
    blksize: 4096,
    ino: 1535785,
    size: 2145232095,
    blocks: 4189912,
    atimeMs: 1614822480703.108,
    mtimeMs: 1614823606226.341,
    ctimeMs: 1614823663578.247,
    birthtimeMs: 0,
    atime: '2021-03-04T01:48:00.703Z',
    mtime: '2021-03-04T02:06:46.226Z',
    ctime: '2021-03-04T02:07:43.578Z',
    birthtime: '1970-01-01T00:00:00.000Z'
  },
  file_size: 2045.8527517318726,
  ffProbeData: { streams: [ [Object], [Object], [Object] ] },
  meta: {
    SourceFile: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    errors: [],
    Duration: 2531.116,
    DefaultDuration: 0.032,
    ExifToolVersion: 12.1,
    FileName: "The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv",
    Directory: 'V:/The Flash (2014)/Season 7',
    FileSize: '2046 MB',
    FileModifyDate: {
      year: 2021,
      month: 3,
      day: 4,
      hour: 2,
      minute: 6,
      second: 46,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:04 02:06:46+00:00'
    },
    FileAccessDate: {
      year: 2021,
      month: 3,
      day: 4,
      hour: 1,
      minute: 48,
      second: 0,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:04 01:48:00+00:00'
    },
    FileInodeChangeDate: {
      year: 2021,
      month: 3,
      day: 4,
      hour: 2,
      minute: 7,
      second: 43,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:04 02:07:43+00:00'
    },
    FilePermissions: 'rw-r--r--',
    FileType: 'MKV',
    FileTypeExtension: 'mkv',
    MIMEType: 'video/x-matroska',
    EBMLVersion: 1,
    EBMLReadVersion: 1,
    DocType: 'matroska',
    DocTypeVersion: 2,
    DocTypeReadVersion: 2,
    TimecodeScale: '1 ms',
    MuxingApp: 'libebml v1.3.3 + libmatroska v1.4.4',
    WritingApp: "mkvmerge v9.0.1 ('Obstacles') 64bit",
    DateTimeOriginal: {
      year: 2021,
      month: 3,
      day: 3,
      hour: 23,
      minute: 21,
      second: 19,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2021:03:03 23:21:19Z'
    },
    VideoCodecID: 'V_MPEG4/ISO/AVC',
    VideoFrameRate: 23.976,
    TrackLanguage: 'und',
    ImageWidth: 1920,
    ImageHeight: 1080,
    DisplayWidth: 1920,
    DisplayHeight: 1080,
    AudioCodecID: 'A_AC3',
    AudioSampleRate: 48000,
    AudioChannels: 6,
    TrackNumber: 3,
    TrackType: 'Subtitle',
    TrackDefault: 'No',
    CodecID: 'S_TEXT/UTF8',
    ImageSize: '1920x1080',
    Megapixels: 2.1
  },
  mediaInfo: { '@ref': '', track: [ [Object], [Object], [Object], [Object] ] },
  hasClosedCaptions: false,
  DB: 'CSwLJ736O',
  bumped: false,
  HealthCheck: '',
  TranscodeDecisionMaker: '',
  processingStatus: false,
  fileMedium: 'video',
  infoLog: '',
  video_codec_name: 'h264',
  audio_codec_name: '',
  video_resolution: '1080p',
  lastHealthCheckDate: 0,
  lastTranscodeDate: 0,
  history: '',
  oldSize: 0,
  newSize: 0,
  tPosition: 0,
  hPosition: 0
}
{
  container: '.mkv',
  FFmpegMode: true,
  handBrakeMode: false,
  infoLog: '☑ Multi channel audio already exists in AC3\n' +
    '☑ No audio processing necessary\n' +
    '☒ Transcoding to HEVC using NVidia NVENC\n' +
    'Encoder configuration:\n' +
    '• Original Bitrate: 6466\n' +
    '• Target Bitrate: 3694\n' +
    '• Minimum Bitrate: 2585\n' +
    '• Maximum Bitrate: 4802\n' +
    '\n' +
    '☑ No subtitle processing necessary',
  processFile: true,
  preset: '-c:v h264_cuvid,-map 0 -map -0:d -c:v hevc_nvenc -rc:v vbr_hq -cq:v 19 -b:v 3694k -minrate 2585k -maxrate 4802k -bufsize 6466k -spatial_aq:v 1 -rc-lookahead:v 32 -c:a copy -c:s copy -max_muxing_queue_size 4096',
  reQueueAfter: false,
  lastPluginDetails: {
    source: 'Community',
    id: 'Tdarr_Plugin_drdd_standardise_all_in_one',
    number: '2/2'
  },
  ffmpegMode: true
}
[2021-03-06T10:37:02.541] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T10:37:02.544] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T10:37:02.545] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T10:37:02.549] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T10:37:02.549] [INFO] Tdarr_Node - verbose:Worker exit request received
[2021-03-06T10:37:02.550] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T10:37:02.562] [INFO] Tdarr_Node - verbose:Worker exited
[2021-03-06T11:00:57.864] [INFO] Tdarr_Node - verbose:Sub-worker exit status received
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-nodeToServer---------
 PT (W:) -> (/mnt/transcode_cache):W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv -> /mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv
 PT (W:) -> (/mnt/transcode_cache):W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv -> /mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv
 Obj translated from:
{
  file: {
    _id: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
    file: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv"
  }
}
 To:
{
  file: {
    _id: "/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
    file: "/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv"
  }
}

 ----------------------------------
[2021-03-06T11:00:57.887] [INFO] Tdarr_Node - verbose:Sub-worker exited
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 PT (/mnt/transcode_cache) -> (W:):/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv -> W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv
 PT (/mnt/transcode_cache) -> (W:):/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv -> W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv
 Obj translated from:
{
  _id: "/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
  file: "/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv"
}
 To:
{
  _id: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
  file: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv"
}

 ----------------------------------
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
File is not video
 ---------Translating-nodeToServer---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
{
  _id: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
  file: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
  container: 'mkv',
  ffProbeRead: '',
  createdAt: 1615053658099,
  lastPluginDetails: 'none',
  bit_rate: 0,
  statSync: { mtimeMs: 0, ctimeMs: 0, ctime: '', mtime: '', atime: '' },
  file_size: 0,
  ffProbeData: { streams: [] },
  meta: {},
  mediaInfo: {},
  hasClosedCaptions: false,
  DB: 'CSwLJ736O',
  bumped: false,
  HealthCheck: '',
  TranscodeDecisionMaker: '',
  processingStatus: false,
  fileMedium: 'other',
  infoLog: "FFprobe was unable to extract data from this file. It is likely that the file is corrupt else FFprobe can't handle this file.",
  video_codec_name: '',
  audio_codec_name: '',
  video_resolution: '',
  lastHealthCheckDate: 0,
  lastTranscodeDate: 0,
  history: '',
  oldSize: 0,
  newSize: 0,
  tPosition: 0,
  hPosition: 0
}
{
  container: '.mkv',
  FFmpegMode: true,
  handBrakeMode: false,
  infoLog: '☑ No multi channel audio found\n' +
    '☑ No audio processing necessary\n' +
    '☑ No video processing necessary\n' +
    '☑ No subtitle processing necessary\n' +
    '☑ No need to process file',
  processFile: false,
  preset: ',-map 0 -map -0:d -c:v copy -c:a copy -c:s copy -max_muxing_queue_size 4096',
  reQueueAfter: false,
  ffmpegMode: true
}
 ---------Translating-nodeToServer---------
 PT (W:) -> (/mnt/transcode_cache):W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv -> /mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv
 PT (W:) -> (/mnt/transcode_cache):W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv -> /mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv
 PT (V:) -> (/mnt/tv):V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv -> /mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv
 Obj translated from:
{
  obj: {
    lastCompletedCacheFile: {
      _id: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
      file: "W:The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv"
    },
    _id: "V:/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}
 To:
{
  obj: {
    lastCompletedCacheFile: {
      _id: "/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv",
      file: "/mnt/transcode_cacheThe Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper-TdarrCacheFile-zYI_kt885.mkv"
    },
    _id: "/mnt/tv/The Flash (2014)/Season 7/The Flash (2014) - S07E01 - All's Wells That Ends Wells HDTV-1080p Proper.mkv"
  }
}

 ----------------------------------
 ---------Translating-serverToNode---------
 Obj translated from:
{}
 To:
{}

 ----------------------------------
[2021-03-06T11:00:58.527] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-06T11:00:58.528] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-06T11:00:58.529] [INFO] Tdarr_Node - verbose:Getting new task
[2021-03-06T11:00:58.535] [INFO] Tdarr_Node - verbose:Exit request
[2021-03-06T11:00:58.535] [INFO] Tdarr_Node - verbose:Worker exit request received
[2021-03-06T11:00:58.537] [INFO] Tdarr_Node - verbose:Exiting
[2021-03-06T11:00:58.546] [INFO] Tdarr_Node - verbose:Worker exited
```

Let me know if you'd like anything more.

**Comment by HaveAGitGat** [comment]
Thanks for that. Would you mind trying the new 2.00.07 package? Should now be fixed. It was a dependency bug with Windows root drive locations.

**Comment by gaggle331** [comment]
Looks like it worked! Thanks for fixing that.

**Comment by HaveAGitGat** [comment]
Glad it's working :)
